### ISSUE TITLE
Console logs are not working correctly

### ISSUE THREAD

**Issue by shanep2300** [angular/protractor#3609]
Hello,
- Node Version: `6.2.1`
- Protractor Version: `4.0.9`

I have these blocks set up in my config file:

```
jasmineNodeOpts: {    
    showColors: true, // If true, print colors to the terminal.      
    print: function() {} // Remove protractor dot reporter.
},

onPrepare: function() {
    var SpecReporter = require('jasmine-spec-reporter');
       jasmine.getEnv().addReporter(new SpecReporter({
       displayStacktrace: 'none',
       displayFailuresSummary: false
    }));
}
```

This was working on 3.2.2 before I updated to 4.0.9. I'm using jasmine-spec-reporter and both jsp and protractor's console logs are showing. I've tried the different NodeOpts to remove the failure summary and the dot reporter is showing next to the jsp steps. It seems to be ignoring the config.

See attachment for what the console log now looks like.

Regards,
Shane

![ss_000043](https://cloud.githubusercontent.com/assets/16820289/18964836/94f38144-8648-11e6-8392-934b7794738d.jpg)

**Comment by cnishina** [comment]
This has something to do when we migrated to the new logger. My initial guess would be to to remove all the logs by setting the static value of the Logger class:

```
var Logger = require('protractor/built/logger').Logger;
var WriteTo = require('protractor/built/logger').WriteTo;

Logger.writeTo = WriteTo.NONE;
```

**Comment by shanep2300** [comment]
For some reason I didn't receive an email or notification when you replied... sorry for the long delay in response. Also, added the JSP block to the OP, as I previously forgot to show that.

 I added what you mentioned and now I didn't get any report but the dot report.
![image](https://cloud.githubusercontent.com/assets/16820289/19453172/1a83c728-9482-11e6-95ae-4d0a1d432e2c.png)

This might also be a JSP issue and not necessarily Protractor. Should I transfer the issue JSP?

**Comment by cnishina** [comment]
Investigating this in more detail: it appears that we introduced Jasmine 2.5.2 in September. The version upgrade caused the issue. The versions that have been affected by this include Protractor 4.0.9 and 4.0.10.

**Comment by cnishina** [comment]
The commit that broke this feature is: https://github.com/jasmine/jasmine-npm/commit/5989948cafa5479f7bde23ddced2b8a3604d003a

There is an issue already open for this: https://github.com/jasmine/jasmine-npm/issues/95

Plan to put a PR to fix this.

**Comment by cnishina** [comment]
Pull request https://github.com/jasmine/jasmine-npm/pull/98

**Comment by mgiambalvo** [comment]
This should be fixed in 4.0.11 - we're downgrading the version of Jasmine we use until they fix the problem on their end.

**Comment by cnishina** [comment]
Closed with https://github.com/jasmine/jasmine-npm/pull/98
